simconvoi.h

```diff
@@ -412,7 +412,10 @@ class convoi_t : public sync_steppable, public overtaker_t
 	void set_erstes_letztes();
 
 	// returns the index of the vehikel at position length (16=1 tile)
-	int get_vehicle_at_length(uint16);
+	int get_vehicle_at_length(sint32);
+
+	// returns the total vehicle number of the coupling convoys.
+	int get_total_vehicle_length();
```

Comment:
`int` はSimutransでは推奨されません。OSによってintが16bit整数だったり32bit整数だったりするためです。
lengthだと1タイルあたり16lengthを想起させるので、車両数であればcountが単語として適切です。
```suggestion
	// returns the total vehicle count including the coupling convoys.
	uint16 get_total_vehicle_count();
```